android/src/main/java/com/mattermost/networkclient/helpers/Helper.kt

```diff
@@ -73,18 +79,12 @@ fun Request.Builder.parseOptions(options: ReadableMap?, session: OkHttpClient.Bu
 fun OkHttpClient.Builder.parseOptions(options: ReadableMap?, request: Request.Builder?): OkHttpClient.Builder {
     if(options == null) return this;
 
-    // Following Redirects
-    if (options.hasKey("followRedirects")) {
-        val followRedirects = options.getBoolean("followRedirects")
-        this.followRedirects(followRedirects)
-        this.followSslRedirects(followRedirects)
-    }
-
-    // Retries
+    // Retries off by default
     this.retryOnConnectionFailure(false);
+
+    // Set retry policy if provided
     if (options.hasKey("retryPolicyConfiguration")) {
```

Comment:
Should you add the linear retry in this PR or is it in a separate one?

Reply:
I have it here separately already: https://github.com/mattermost/react-native-network-client/pull/53